benchmarks/perf-tool/release-configs/faiss-hnsw/filtering/relaxed-filter/relaxed-filter-test.yml

```diff
@@ -0,0 +1,35 @@
+endpoint: "navneev.aka.corp.amazon.com"
+port: 9900
```

Comment:
Also, the port

Reply:
port I will remove

Reply:
If we are using ELB endpoint and if I send the port(80) with endpoint, it doesn't work right ? We need to set it separately like you added before

Reply:
Yes, we need to set separately.

Reply:
So, shall we add the placeholder to set `port` in all .yml files ?

Reply:
It's fine. not sure how many times we use ELB. Plus I have added things in documentation so we should be good.